src/game.cpp

```diff
@@ -1062,6 +1063,10 @@ void Game::run()
 	irr::core::dimension2d<u32> previous_screen_size(g_settings->getU16("screen_w"),
 		g_settings->getU16("screen_h"));
 
+	Discord::create();
```

Comment:
you must register ENABLE_DISCORD in cmake_config.h.in and use the corresponding ifdef in this source code